Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private string _name;
+        public string GetName()
+        {
+            return _name;
+        }
+        public void SetName(string name)
+        {
+            _name = name;
+        }
+
+        private decimal[] _balances;
+        public decimal[] GetBalances()
+        {
+            return _balances;
+        }
+        public void SetBalances(decimal[] balances)
+        {
+            _balances = balances;
+        }
+
+        public decimal GetCurrentBalance()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetCurrentBalance' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)

Reply:
I thought that  we were supposed to use Get and Set methods.  I don't recall you going over Properties (but maybe missed that part).  Am I suppose to use Properites?